mitodl/micromasters
# Added page for terms of service

#### What are the relevant tickets?
Fixes #178 

#### What's this PR do?
Adds a required terms of service page. When the user clicks 'Accept' the user will be directed to the profile. If they click 'Cancel' the user will be logged out and redirected to the homepage. There is also a new field in the REST API and model to keep track of whether a user agreed to this.

Since there are UI tests here there is a bit of new code to handle this testing. The routes are also moved into `dashboard_routes.js`. 

#### Where should the reviewer start?
static/js/containers/TermsOfServicePage.js

#### How should this be manually tested?
You should be forced to go to the terms of service page whenever you are logged in and are in the dashboard area. When you agree to the terms of service (currently blank) you should be allowed to go anywhere.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
